File: Template.CSharp/Contract1.cs
using Neo.SmartContract.Framework;
using Neo.SmartContract.Framework.Services.Neo;
using System;
using System.Numerics;

namespace $safeprojectname$
{
	public class Contract1 : SmartContract
{
    public static bool Main(string operation, object[] args)
    {
        Storage.Put("Hello", "World");
        return true;
    }
}
}

File: templates/visual-studio-vsix/Template.VB/Contract1.vb
Imports Neo.SmartContract.Framework
Imports Neo.SmartContract.Framework.Services.Neo
Imports System
Imports System.Numerics

Public Class $itemname$ : Inherits SmartContract
    Public Shared Function Main(ByVal operation As String, ByVal args() As Object) As Boolean
        Storage.Put("Hello", "World")
        Return True
    End Function
End Class
